tests/torchtune/data/test_collate.py

```diff
@@ -8,7 +8,7 @@
 
 import torch
 
-from torchtune.utils.collate import padded_collate
+from torchtune.data._collate import padded_collate
```

Comment:
super nit: not sure if you need the full namespace
```suggestion
from torchtune.data import padded_collate
```

Reply:
I am totally burned by all the circular imports, so I'm going to be explicit in most cases